model/messaging/registry.yaml

```diff
@@ -33,9 +33,12 @@ groups:
         type: string
         stability: experimental
         brief: 'The message destination name'
-        note: |
+        note: >
           Destination name SHOULD uniquely identify a specific queue, topic or other entity within the broker. If
           the broker doesn't have such notion, the destination name SHOULD uniquely identify the broker.
+
+          If `messaging.destination.anonymous` or `messaging.destination.temporary` is set to true,
```

Comment:
we still want to capture the actual destination name, don't we? 

Temporary queues (e.g. https://www.rabbitmq.com/docs/queues#temporary-queues) are not different from regular queues, but they are short lived and their names might have high cardinality.

E.g. rabbit broker can generate random(?) name - https://www.rabbitmq.com/docs/queues#server-named-queues or user might assign it.

In any case, it seems important to capture the actual queue name regardless of its life expectancy on spans, but protect span name and metrics from high-cardinality.

Reply:
Do we have alternatives if we want to capture the destination name, other than "hiding" it behind a config option? (default would be using "(temporary)")

Reply:
> In any case, it seems important to capture the actual queue name regardless of its life expectancy on spans, but protect span name and metrics from high-cardinality.

Right I think that makes sense on the `destination.name` here. So, what is really missing is define what to put on the span name when the destination is temporary/anonymous. 

I removed this now and tweaked the "destination" concept for span name. PTAL